engine/src/main/scala/cromwell/server/CromwellSystem.scala

```diff
@@ -5,13 +5,13 @@ import akka.http.scaladsl.Http
 import akka.stream.ActorMaterializer
 import com.typesafe.config.ConfigFactory
 import cromwell.engine.backend.{BackendConfiguration, CromwellBackends}
-import cromwell.services.SingletonServicesStore
+import cromwell.services.{MetadataServicesStore, OperationsServicesStore}
 
 import scala.concurrent.Future
 
 trait CromwellSystem {
   /*
-  Initialize the services-store (aka a collection of DAOs), if they haven't been already, before starting any services
+  Initialize the services-stores (aka a collection of DAOs), if they haven't been already, before starting any services
```

Comment:
I realize you just pluralized this here but this seems funky. I suspect the issue is just with how the rest of the comment is phrased. I think what the comment was trying to get at was that the services in the service registry were being kicked off? TBH I'm not really sure, but it seems like something either about the wording or what's actually happening isn't accurate